### ISSUE TITLE
fail to import latest lighthouse packages from stable

### ISSUE THREAD

**Issue by zhiqiangxu** [sigp/lighthouse#5337]
After updating lighthouse packages to latest stable branch, I come across these errors:

```
error[E0433]: failed to resolve: could not find `u256_dec` in `serde_utils`
    --> /Users/alanxu/.cargo/git/checkouts/lighthouse-d93a8d887484268e/7b65d38/common/eth2/src/types.rs:1561:20
     |
1561 |     #[serde(with = "serde_utils::u256_dec")]
     |                    ^^^^^^^^^^^^^^^^^^^^^^^ could not find `u256_dec` in `serde_utils`

error[E0433]: failed to resolve: could not find `u256_dec` in `serde_utils`
    --> /Users/alanxu/.cargo/git/checkouts/lighthouse-d93a8d887484268e/7b65d38/common/eth2/src/types.rs:1563:20
     |
1563 |     #[serde(with = "serde_utils::u256_dec")]
```

It seems `serde_utils::u256_dec` only shows up twice in the codebase, are they necessary?

Step to reproduce:

```bash
git clone https://github.com/zhiqiangxu/eth-tool
cd eth-tool
cargo build
```

**Comment by michaelsproul** [comment]
yes it's necessary, try running `cargo update -p ethereum_serde_utils`

**Comment by zhiqiangxu** [comment]
@michaelsproul Thanks, it works! Why doesn't it work out of the box though? And what does `-p` option do? I didn't find it in the help:)

**Comment by michaelsproul** [comment]
`-p` updates a single package in the Cargo.lock file

The root cause is that we incremented the patch version of ethereum_serde_utils when adding that feature but didn't bump the min version in Lighthouse's Cargo.toml. We did bump it in the Cargo.lock, which is why Lighthouse compiles fine.

If you deleted your Cargo.lock (or started from scratch) then your project would also compile.

The fix is to bump the version in Lighthouse's Cargo.toml so that Cargo never attempts to compile Lighthouse against the old version of ethereum_serde_utils that is missing the new feature
